libsolidity/parsing/DocStringParser.cpp

```diff
@@ -39,12 +40,19 @@ string::const_iterator skipLineOrEOS(
 	return (_nlPos == _end) ? _end : ++_nlPos;
 }
 
-string::const_iterator firstSpaceOrTab(
+string::const_iterator firstNonIdentifier(
 	string::const_iterator _pos,
 	string::const_iterator _end
 )
 {
-	return boost::range::find_first_of(make_pair(_pos, _end), " \t");
+	auto currPos = _pos;
+	if (currPos == _pos && isIdentifierStart(*currPos))
```

Comment:
Why isn't just the `while` loop enough?

Reply:
Because identifiers are not allowed to start with a digit and that's why I'm checking the first character before continuing.